tasks/main.yaml

```diff
@@ -14,6 +14,16 @@
       - podman
     state: present
 
+- name: Create an empty mounts.conf to override podman default mounts
+  # See also: man containers-mounts.conf (shipped by containers-common)
```

Comment:
Can we also add comment here that the main purpose is to avoid having host repos injected into OSBS image builds?

Reply:
sure, pr is rebased and added comment for this.